models/models.js

```diff
@@ -42,4 +42,36 @@ exports.fetchComments = (article_id) => {
       }
   return rows
    })
+}
+
+exports.addComment = (comment, article_id) => {
+   return db.query(`SELECT * FROM users WHERE username = $1`, [comment.author])
+   .then(({rows}) => {
+      if (rows.length){
+         return db.query(`INSERT INTO comments (body, author, article_id) VALUES ($1, $2, $3) RETURNING *;`, [comment.body, comment.author, article_id])
+         .then(({rows}) => {
+            return rows[0];
+         })
+      }
+      else {return Promise.reject({
+         status: 400,
+         message: 'bad request'
+      })}
+   })
+}
+
+exports.updateArticle = (num, article_id) => {
+   return db.query(`SELECT * FROM articles WHERE article_id = $1;`, [article_id])
```

Comment:
Nice that you're checking the article exists, do you have a model that you could call instead of writing out the SQL query again?